tracer/build/_build/Build.cs

```diff
@@ -219,6 +221,8 @@ void DeleteReparsePoints(string path)
         .DependsOn(PublishManagedTracerR2R)
         .DependsOn(DownloadLibDdwaf)
         .DependsOn(CopyLibDdwaf)
+        .DependsOn(DownloadLibDatadog)
+        .DependsOn(CopyLibDatadog)
```

Comment:
This raises an interesting question actually - _should_ we enable the trace exporter in the AWS Lambda artifacts 🤔 My gut feeling is no, at least initially, as they're very sensitive to size. It's also possible that the lambda layer would strip out this file anyway!

Reply:
I will follow you gut on that

Reply:
The plan was to cover AWS Lambda as well, Lambda layer I think allows 250MB and specially these are all AL2 instances - we should be good.

My other question related to this, do we test on AL2 instances in our CI?

Reply:
(this PR is merged already, but commenting here for visiblity)

> should we enable the trace exporter in the AWS Lambda artifacts? My gut feeling is no, at least initially, as they're very sensitive to size. It's also possible that the lambda layer would strip out this file anyway!

Yup, we are already stripping it out (https://github.com/DataDog/dd-trace-dotnet-aws-lambda-layer/pull/10), so including this in the Lambda artifacts is useless at the moment.